azure-iot-device/tests/common/pipeline/conftest.py

```diff
@@ -0,0 +1,7 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+pytest_plugins = ["tests.common.pipeline.fixtures"]
```

Comment:
This is throwing warnings for me.

Reply:
I think you're right.  I will change this.  

---
In reply to: [291668287](https://github.com/Azure/azure-iot-sdk-python-preview/pull/104#discussion_r291668287) [](ancestors = 291668287)